### DISCUSSION TITLE
Connector at J202

### DISCUSSION THREAD

**Discussion by jhanssen** [ZuluIDE/ZuluIDE-firmware#108] [Q&A]
Hi, I recently got one of these (2024a) and I'm wondering what the three pin connector at J202 is for? Is it the debug port (SWCLK / SWDIO)?

Thanks.

**Comment by aperezbios** [comment]
@jhanssen apologies for the delayed response. I missed this.

Yes, J202 is an SWD debug header, intended for use with the official https://www.raspberrypi.com/documentation/microcontrollers/debug-probe.html, which can be had for around $12 from Mouser and other distributors. It is a standard CMSIS-DAP compatible device, and works with OpenOCD. 

Also see https://datasheets.raspberrypi.com/debug/debug-connector-specification.pdf

The adjacent J301 is a full ARM Cortex-M debug header, for the pinout there, see https://developer.arm.com/documentation/101416/0100/Hardware-Description/Target-Interfaces/Cortex-Debug--10-pin-
